flink-connector-http/src/main/java/org/apache/flink/connectors/http/internal/sink/httpclient/RequestSubmitterFactory.java

```diff
@@ -0,0 +1,29 @@
+/*
+ *
+ *  * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
The license comment looks strange to me. Are those two columns consisting of `*` expected?